### ISSUE TITLE
Plotting interaction network

### ISSUE THREAD

**Issue by mpm896** [KosinskiLab/AlphaPulldown#141]
Hello,
I have a question about how to visualize a large interaction network. For example, I've run >1000 predictions on various proteins and want to be able to parse through the data and visualize the interaction network, but it's a bit overwhelming. I was inspired by one of the talks at the CSSB conference where Gephi was used to accomplish this by plotting the interactions based on the PAE, but I'm not sure where to begin and would really appreciate some advice! Thank you.

Matt

**Comment by jkosinski** [comment]
Yes, we will add this feature at some point. In the meantime, you could try to run the following script that formats the CSV file created by AlphaPulldown analysis step to a Cytoscape network format:
```python
import sys
import csv

interactions = []
with open(sys.argv[1]) as f:
    reader = csv.DictReader(f)
    for row in reader:
        job = row['jobs']
        if '_and_' in job and float(row['iptm_ptm']) > 0.6:
            items = job.split('_and_')
            names = items
            score = row['iptm_ptm']
            if names not in [x[0] for x in interactions]:
                interactions.append([names, score])

with open(sys.argv[2], 'w') as f:
    f.write('source,target,interaction,score\n')
    for inter, score in interactions:
        f.write(f'{inter[0]},{inter[1]},pp,{score}\n')
```
It takes two arguments, first input CsV and second output CSV. 0.6 is the confidence threshold you should adjust.

Then you can import to Cytoscape like this:
https://manual.cytoscape.org/en/stable/Creating_Networks.html#import-networks-from-unformatted-table-files

In Cytoscape you can adjust edge color or width to the confidence score.

Feel free to leave this issue open to remind us this feature request.
